File: Golang/Easy/1716_Calculate_Money_In_Leetcode_Bank/solution.go
package calculatemoneyinleetcodebank

func totalMoney(n int) int {
	start := 1
	count := 0

	for i := 1; i <= n; i++ {
		count += start
		start += 1

		if i%7 == 0 {
			start = start - 7 + 1
		}
	}
	return count
}

File: Python/Easy/1716_Calculate_Money_In_Leetcode_Bank/solution.py
class Solution:
    def totalMoney(self, n: int) -> int:
        start = 1
        count = 0

        for i in range(1, n+1):
            count += start
            start += 1

            if i % 7 == 0:
                start = start - 7 + 1

        return count